README.md

```diff
@@ -103,11 +103,11 @@ bg := `{{ Background "#0000ff" "Blue Background" }}`
 wrap := `{{ Bold (Underline "Hello World") }}`
 
 // parse and render
-tpl = tpl.Parse(bold)
+tpl, _ = tpl.Parse(bold)
 
 var buf bytes.Buffer
 tpl.Execute(&buf, nil)
-fmt.Println(buf)
+fmt.Println(buf.String())
```

Comment:
That's actually not necessary. The `String()` function will automatically be called by `Println` when available.

Reply:
Good to know. I'm definitely not a go-expert. But this code

    f := termenv.TemplateFuncs(termenv.ColorProfile())
	tpl := template.New("tpl").Funcs(f)
	crossout := `{{ CrossOut "Blablablabla" }}`
	tpl, _ = tpl.Parse(crossout)
	var buf bytes.Buffer
	tpl.Execute(&buf, nil)
	fmt.Println(buf)

prints

    {[27 91 57 109 66 108 97 98 108 97 98 108 97 98 108 97 27 91 48 109] 0 0}

Reply:
But the better solution would then probably be

    fmt.Println(&buf)